enzyme/Enzyme/CMakeLists.txt

```diff
@@ -44,32 +44,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 list(APPEND ENZYME_SRC SCEV/ScalarEvolutionExpander.cpp)
 list(APPEND ENZYME_SRC TypeAnalysis/TypeTree.cpp TypeAnalysis/TypeAnalysis.cpp TypeAnalysis/TypeAnalysisPrinter.cpp TypeAnalysis/RustDebugInfo.cpp)
 
-if (${LLVM_VERSION_MAJOR} LESS 8)
```

Comment:
Do we want to add a cmake assertion that LLVM_VERSION_MAJOR >= 9?

Reply:
done